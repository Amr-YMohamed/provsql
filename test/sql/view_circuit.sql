SET search_path TO public, provsql;

CREATE TABLE vc_result AS
SELECT city, view_circuit(provenance(),'d',1) AS prob
FROM (
  SELECT DISTINCT city
  FROM personal
EXCEPT 
  SELECT p1.city
  FROM personal p1,personal p2
  WHERE p1.id<p2.id AND p1.city=p2.city
  GROUP BY p1.city
) t;

select * from vc_result;
