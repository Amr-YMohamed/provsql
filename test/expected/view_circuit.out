\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

   city   | res 
----------+-----
 Berlin   |   0
 New York |   0
 Paris    |   0
(3 rows)

